lib/config/validation.ts

```diff
@@ -164,22 +177,27 @@ export async function validateConfig(
       });
     }
 
-    if (configType === 'global' && isGlobalOption(key)) {
-      await validateGlobalConfig(
-        key,
-        val,
-        optionTypes[key],
-        warnings,
-        currentPath,
-      );
-      continue;
-    } else {
-      if (isGlobalOption(key) && !isFalseGlobal(key, parentPath)) {
-        warnings.push({
-          topic: 'Configuration Error',
-          message: `The "${key}" option is a global option reserved only for Renovate's global configuration and cannot be configured within repository config file.`,
-        });
+    if (isGlobalOption(key)) {
+      if (configType === 'global') {
+        await validateGlobalConfig(
+          key,
+          val,
+          optionTypes[key],
+          warnings,
+          currentPath,
+        );
         continue;
+      } else {
+        if (
+          !isFalseGlobal(key, parentPath) &&
+          !(configType === 'inherit' && isInhertConfigOption(key))
+        ) {
+          warnings.push({
+            topic: 'Configuration Error',
+            message: `The "${key}" option is a global option reserved only for Renovate's global configuration and cannot be configured within repository config file.`,
```

Comment:
```suggestion
            message: `The "${key}" option is a global option reserved only for Renovate's global configuration and cannot be configured within a repository's config file.`,
```

The message feels a bit ungrammatical. Is this better?